tools/devservices_healthcheck.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+import os
+import subprocess
+from time import sleep
+
+
+def run_cmd(
+    args: list[str],
+    *,
+    retries: int = 3,
+    timeout: int = 5,
+) -> int:
+    for retry in range(1, retries + 1):
+        try:
+            returncode = subprocess.call(args)
+        except OSError:
+            raise SystemExit(1)
```

Comment:
I meant that this should bubble up as an error -- we shouldn't catch this since it's a _programming_ error not a health error